src/main/resources/resources.properties

```diff
@@ -125,7 +125,12 @@ ui.authenticationView.noEmptyToken=The token cannot be empty
 ui.authenticationView.tokenLink=What is my token?
 
 ### Misc
-ui.aboutDialog.description=<html>The A+ Courses plugin supports the educational use of IntelliJ (and its Scala plugin) in<br>programming courses that rely on the A+ learning management system, which has been<br>developed at Aalto University. The plugin accesses programming assignments<br>and automated grading services provided by A+ and otherwise enhances<br>the student experience.</html>
-ui.aboutDialog.title=About the A+ Courses Plugin
+ui.aboutDialog.description=<html>The A+ Courses plugin supports the educational use of IntelliJ (and its Scala plugin)<br>in programming courses that rely on the A+ learning management system, which<br>has been developed at Aalto University.<br><br>The plugin accesses programming assignments and automated grading services<br>provided by A+ and otherwise enhances the student experience.</html>
+ui.aboutDialog.title=A+ Courses
 ui.aboutDialog.website=A+ Courses Plugin website
 ui.localHistory.submission.tag={0}, {1}, Submission {2}
+ui.aboutDialog.version=<html><b>Version: {0}</b></html>
+ui.aboutDialog.authors=<html>Developed by the department of Computer Science at Aalto University.</html>
```

Comment:
The `<html>` tags are most likely unnecessary in this line, since there's no styling like `<b>` or anything